src/states/render-messages.ts

```diff
@@ -198,6 +198,35 @@ export const renderAmizoneMenu = () => ({
   },
 });
 
+export const renderReplyMessage = () => ({
```

Comment:
Rename this to `renderQuickAttendanceButtons`, or something descriptive like that.